app/components/page_settings_summary_component/view.rb

```diff
@@ -16,14 +16,24 @@ def initialize(page_object, change_answer_type_path: "", change_selections_setti
   private
 
     def address_input_type_to_string
-      input_type = @page_object.answer_settings.input_type
-      if input_type.uk_address == "true" && input_type.international_address == "true"
+      input_type = settings[:input_type]
+      if input_type[:uk_address] == "true" && input_type[:international_address] == "true"
         "uk_and_international_addresses"
-      elsif input_type.uk_address == "true"
+      elsif input_type[:uk_address] == "true"
         "uk_addresses"
       else
         "international_addresses"
       end
     end
+
+    def settings
```

Comment:
We should probably call this `answer_settings` to avoid confusion with `Settings`